### ISSUE TITLE
Clarify resampling on beta images vs ROIs

### ISSUE THREAD

**Issue by emdupre** [courtois-neuromod/cneuromod-things#45]
I'm having a little bit of trouble tracking down the point at which the fLoc defined ROIs 
> Shape (208, 300, 320) 

diverge from the trial-averaged beta images 
> Shape (76, 90, 71)

Is this noted somewhere in the documentation ?

**Comment by emdupre** [comment]
I assume it's that the fLoc are acquired at a higher resolution and / or that the target resolution during fMRIPrep is 2mm^3, but given #46  I'm having trouble confirming this

**Comment by MarieStLaurent** [comment]
Does the link to the pipeline and scripts from the main fLoc readme help to resolve this?

https://github.com/courtois-neuromod/cneuromod-things/tree/main/fLoc
https://github.com/courtois-neuromod/cneuromod-things/tree/main/fLoc/rois/code

**Comment by MarieStLaurent** [comment]
Yes, that was not clear. The warping of atlas-derived parcels from MNI to T1w space is done with the subject's anatomical scan as a reference, so the output's resolution is higher than the functional scans. I added the tag "res-anat" in the file name to make it explicit. 

I had not meant for these masks to be used as-is, but I know see their value (especially for subjects who did not complete the fLoc localiser task). I generated another series of `from_atlas` masks that are binarized and whose resolution match the functional scans in native subject space.
E.g., ```
cneuromod-things/fLoc/rois/sub-01/rois
/from_atlas/sub-01_parcel-kanwisher_space-T1w_res-func_contrast-face_roi-FFA_desc-L_mask.nii.gz
```
